music21/test/testArticulations.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import music21
+
+
+class TestArticulations(unittest.TestCase):  # pragma: no cover
+    def testArticulations(self):
```

Comment:
this method should be in the Test class under xmlToM21 -- see the examples of other demos, for instance `testTrillOnOneNote` on how to load the file.